### DISCUSSION TITLE
VAxios 会不会过度封装了

### DISCUSSION THREAD

**Discussion by PedroGuo** [vbenjs/vue-vben-admin#524] [Q&A]
# vue-vben-admin\src\utils\http\axios\Axios.ts
1、文件中看到了对 axios 库再一次进行了封装，感觉没有必要弄那么复杂，有点过度封装的感觉，不知道这样做有什么好处，想问下

**Comment by maicWorkGithub** [comment]
哪里过度了？你先说一下。先说具体的点

**Comment by M69W** [comment]
开发者只是提供一个模板，在这个框架下的对应的请求处理。
完全可以自己改成自己的使用环境使用，
或者自己写一个

**Reply by maicWorkGithub** [reply]
你这还是太笼统了，你觉得有不符合自己使用的地方，自己修改；如果觉得他写的有不好的地方，说出来大家讨论一下，或者直接提个PR
